docker-compose.network.yml

```diff
@@ -164,7 +164,3 @@ services:
     volumes:
       - ./network/health-check.sh:/health-check.sh
     command: /bin/sh health-check.sh
-  driver:
-    build:
-      context: .
-      dockerfile: ./compose/bigchaindb-driver/Dockerfile
```

Comment:
Why removing the python client service now? This was put there for development purposes and may still be useful as we are not complete with the integration work phase.